src/api/modules/login/login.input.ts

```diff
@@ -0,0 +1,20 @@
+import { InputType, Field } from 'type-graphql';
+import { Length, IsEmail, Matches, IsBoolean } from 'class-validator';
+
+@InputType()
+export class LoginInput {
+  @Field()
+  @IsEmail(undefined, { message: 'O email precisa ser um endereço de e-mail válido' })
+  email!: string;
+
+  @Field()
+  @Length(7, undefined, { message: 'A senha precisa ter pelo menos 7 caracteres' })
+  @Matches('(?=.*[a-zA-ZÀ-ÖÙ-öù-ÿĀ-ž]+)(?=.*[0-9]+).*', undefined, {
+    message: 'A senha precisa ter pelo uma letra e um número',
+  })
+  password!: string;
+
+  @Field()
+  @IsBoolean({ message: 'Lembrar login precisa ser verdadeiro ou falso' })
+  rememberMe: boolean = false;
+}
```

Comment:
O `@Field` precisa receber o tipo do campo. Do contrário, ele assume string. Outra prática legal é, em conformidade com o Typescript/Javascript, avisar o schema qual o valor padrão que esse valor assumirá:

```suggestion
  @Field(() => Boolean, { default: false })
  @IsBoolean({ message: 'Lembrar login precisa ser verdadeiro ou falso' })
  rememberMe: boolean = false;
}
```

Reply:
Ah, eu não sabia desses detalhes. Vou ajustar.